proto/v1/user_service.proto

```diff
@@ -31,6 +31,17 @@ service UserService {
     option (bytebase.v1.auth_method) = CUSTOM;
   }
 
+  // Get user stat.
```

Comment:
Maybe we can use actuator info and only returns the active user count (no deleted users, no service account)? Anyway, later change, let's get this big CL in first.

Reply:
Actuator API also LGTM, but we still need the count with state and type, used by the frontend. For example, we need to show total count (all active types) and deleted count (all deleted types) in the user dashboard, and show the total count (active users+service accounts) for subscription seat limit.

Reply:
Let's move on for now. We can consider and change the information available on the UI.